package.json

```diff
@@ -50,6 +50,23 @@
         "language": "flux",
         "path": "./snippets/flux.json"
       }
+    ],
+    "configuration": [
+      {
+        "title": "FLUX",
+        "properties": {
+          "vsflux.enableInfluxDBConnection": {
```

Comment:
Can we hold off on this until the connection has been added back? I'd rather add it when it's functional.